vars/emailUtils.groovy

```diff
@@ -0,0 +1,27 @@
+/**
+ * Requires "Email Notifications Plugin"
+ */
+
+private def sendEmail(sentTo, replyTo, subject, content) {
+
+        if(sentTo != null && !sentTo.isEmpty()) {
+          emailext(body: content, mimeType: 'text/html',
+             replyTo: '$DEFAULT_REPLYTO', subject: subject,
+             to: sentTo, attachLog: true )
+        }
+}
+
+
+def call(parameters = [:]) {
+        def sentTo = parameters.get('sentTo', pipelineVars.emailDefaultSentTo)
+        def replyTo = parameters.get('replyTo', pipelineVars.emailDefaultReplyTo)
+        def subject = parameters.get('subject', pipelineVars.emailDefaultSubject)
+        def content = parameters.get('content', pipelineVars.emailDefaultContent)
+        def extraJobProperties = parameters.get('extraJobProperties', [])
```

Comment:
Since you are setting job properties here, this looks like a function that is running a whole piopeline for you? If the intent is to have this function is actually run a whole pipeline for you, the convention in this repo is to name this file starting with `exec`. For example:

```
execSendEmail.groovy
```

Reply:
If you intend to just call this function from within another pipeline, then I would name this function `sendEmail`, get rid of the `jobProperties` stuff, and then you'd call it from your other pipeline with something like this:

```
emailUtils.sendEmail(args)
```

Reply:
Thanks for the details. Fixed it now.